src/diffusers/pipelines/animatediff/pipeline_animatediff.py

```diff
@@ -71,17 +71,14 @@ def tensor2vid(video: torch.Tensor, processor: "VaeImageProcessor", output_type:
     for batch_idx in range(batch_size):
```

Comment:
Maybe this tensor2vid function could be moved into the utils folder because it has common use across multiple pipelines and there is yet to be a case where the implementation would be different? @ DN6